webapp/locales/de.json

```diff
@@ -40,7 +40,7 @@
       "tagCountUnique": "Benutzer"
     },
     "invites": {
-      "description": "Einladungen sind ein wunderbarer Weg, deine Freund in deinem Netzwerk zu haben …",
+      "description": "Einladungen sind eine wunderbare Möglichkeit, Deine Freunde in Deinem Netzwerk zu haben …",
```

Comment:
It's OK for me now to use the capitalized `Du` here but according to a teacher of journalism, this spelling is antiquated and except for letters it's now common to use lowercase `du`. So for the future, I would prefer that.